src/elements/ComicScroll.tsx

```diff
@@ -0,0 +1,27 @@
+import ComicCard from "../atoms/ComicCard";
+import { IComicData } from "../models/comicData";
+function ComicScroll({
+  heading,
+  comics,
+}: {
+  heading: string;
+  comics: IComicData[];
+}) {
+  return (
+    <>
+      <h4 className="text-medium font-medium">{heading}</h4>
+      <div className="major-cities-scroll bg-white overflow-auto whitespace-nowrap mb-6">
+        {comics.map((comic) => (
+          <a
```

Comment:
assume you want the user to be able to click on this and let it take you to a comic in the future?

Reply:
I wanted to make a comment, but I realise `a` is going to be changed to `Link` when you implement routing

Reply:
Yea, thats the plan. It will be changed when I implement the routing and stuff